Gems/ROS2/Code/Source/ContactSensor/ROS2ContactSensorComponent.cpp

```diff
@@ -9,6 +9,7 @@
 #include "ROS2ContactSensorComponent.h"
 #include <AzCore/Component/ComponentApplicationBus.h>
 #include <AzCore/Debug/Trace.h>
+#include <AzCore/Serialization/EditContext.h>
```

Comment:
This change will be gone from final PR, but fixes compile error present at the branching so it's easier to test.